scipy/stats/_survival.py

```diff
@@ -180,23 +253,31 @@ def ecdf(sample):
     res : `~scipy.stats._result_classes.ECDFResult`
         An object with the following attributes.
 
-        x : ndarray
-            The unique values in the sample.
         cdf : `~scipy.stats._result_classes.EmpiricalDistributionFunction`
-            An object representing the empirical cumulative distribution function.
+            An object representing the empirical cumulative distribution
+            function.
         sf : `~scipy.stats._result_classes.EmpiricalDistributionFunction`
             An object representing the empirical survival function.
 
         The `cdf` and `sf` attributes themselves have the following attributes.
 
-        points : ndarray
-            The point estimate of the CDF/SF at the values in `x`.
+        quantiles : ndarray
```

Comment:
It might come as a surprise that a `cdf` object has a `quantiles` attribute as in `myecdf.cdf.quantiles`. But you already discussed this "point" 😏